src/core/protocols/oauth/oidc_callback.ts

```diff
@@ -104,18 +103,14 @@ export async function OIDCCallback(
     userInfoResponse,
   )
 
-  if (!result.email_verified) {
-    return new UnVerifiedAccountAPIError()
-  }
-
   if (!result.email) {
     return new UnVerifiedAccountAPIError()
   }
 
```

Comment:
We still need to verify if the result.email_verified if we do not skip the verification, I believe.
```typescript
if (!result.email) {
    return new MissingEmailAPIError()
}

if (!skip_email_verification && !result.email_verified) {
    return new UnVerifiedAccountAPIError()
}
```

Reply:
Well, I think we got lost in the process... That would work for Microsoft Entra, but this provider is not the only provider that uses OpenID. That *may* break other providers?

Reply:
Sounds good to me!